tests/integration/framework/kill/kill_windows.go

```diff
@@ -0,0 +1,55 @@
+//go:build windows
+// +build windows
+
+/*
+Copyright 2023 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package kill
+
+import (
+	"os/exec"
+	"syscall"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"golang.org/x/sys/windows"
+)
+
+func interrupt(t *testing.T, cmd *exec.Cmd) {
```

Comment:
just a question. does this work as expected, has this been validated?

does `taskkill` work? https://learn.microsoft.com/en-us/windows-server/administration/windows-commands/taskkill

Reply:
I don't really have access to a windows machine so difficult for me to get quick round trip validations done. Very happy to use taskkill instead if that is the more canonical way of doing this.

Reply:
After playing around with `taskill`, it seems that it does not gracefully shutdown the processes.

I'll revert back to the `CTRL_BREAK_EVENT` approach.